ios/Classes/mopro.swift

```diff
@@ -455,7 +455,7 @@ private struct FfiConverterData: FfiConverterRustBuffer {
     }
 }
 
-public struct G1 {
+public struct G1: Codable {
```

Comment:
@vivianjeng I have made the structs serializable here in Swift and also in Kotlin. This simplifies the communication between the individual platform (iOS/Android) and Flutter code because we can just pass a JSON. This change is now applied only to this example project, but I think that we can introduce a "-flutter" compilation option in the main mopro repo that would conditionally add the serialization-related code.

Reply:
Actually the file is auto generated with [uniffi](https://github.com/mozilla/uniffi-rs)
Do you think to change the mopro only (e.g. to add like `Codable` to `mopro.swift`)
or we can change the `uniffi-rs`?

Reply:
If we don't need to re-generate `mopro.swift` and `mopro.kt` often, we can use this https://crates.io/crates/askama to add some conditional code to it.

Reply:
Or parse the generated files and add the necessary code by directly writing in the specific lines.